testdata/globalconfig.yml

```diff
@@ -2,16 +2,16 @@ subject: "namespace:ns"
 revision: "2022"
 adapters:
   - name: default
-    kind: istio/quota
-    impl: istio/memQuota
+    kind: quotas
```

Comment:
With an adapter providing multiple aspects, kind field should be removed from the proto.